invenio_access/permissions.py

```diff
@@ -169,3 +176,15 @@ def excludes(self):
         """
         self._load_permissions()
         return self._permissions.excludes
+
+
+class Permission(DynamicPermission):
```

Comment:
Can we reverse it, so that Dynamic Permission inherits from Permission instead....this will make it easier to remove.

Could be done with:
```python
class Permission(_Permission):
   allow_by_default = False

   def _load_permission(self):
       if not self._allow_by_default:
          self._permissions.needs.update(self.explicity_needs)
       #....


class DynamicPermission(Permission)
     allow_by_default = True

     def __init__(...):
         warnings.warn("DynamicPermission will scheduled for removal", PendingDeprecationWarning)